src/utils/packageInfoFormatter.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+/**
+ * Creates a standardized package info object that can be safely used with JSON.stringify
+ * and in the otpVerification method
+ * @param {string} version - The package version
+ * @param {string} packageName - The package name
+ * @returns {Object} Formatted package info object
+ */
```

Comment:
let's avoid all these docs. docs are longer than the actual code, and this is for internal use only